.github/workflows/bot-nightly.yaml

```diff
@@ -0,0 +1,64 @@
+# Nightly jobs run by a bot collaborator.
+name: Nightly jobs
+on:
+  repository_dispatch:
+    types: [nightly]
+
+jobs:
+  snapshot-source:
+    name: Update Keras guides
+    if : ${{ github.actor == 'tfdocsbot' }}
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        repository: keras-team/keras-io
+        path: keras-io
+    - uses: actions/checkout@v2
+      with:
+        ref: snapshot/keras
+        fetch-depth: 0
+        path: docs
+    - name: Set up repo
+      run: |
+        # Set commit author.
+        git config --global user.name "$GITHUB_ACTOR"
+        git config --global user.email "$GITHUB_ACTOR@users.noreply.github.com"
+        # Update branch.
+        cd docs
+        if ! git merge origin/master; then
+          echo "[${GITHUB_WORKFLOW}] Unable to update from master branch." >&2
+          exit 1
+        fi
+    - name: Set up Python
+      uses: actions/setup-python@v2
+    - name: Install requirements
+      run: |
+        python3 -m pip install -U -r keras-io/requirements.txt
+        python3 -m pip install -U git+https://github.com/tensorflow/docs
+    - name: Generate Keras notebooks
+      run: |
+        cd keras-io/scripts/
+        echo "[${GITHUB_WORKFLOW}] Generate Keras guides ..."
+        python3 autogen.py generate_tf_guides  # Output to ../tf/
+        echo "[${GITHUB_WORKFLOW}] Format notebooks ..."
+        python3 -m tensorflow_docs.tools.nbfmt ../tf/
+    - name: Sync docs repo
+      env:
+        KERAS_GUIDES_DIR: site/en/guide/keras/
+      run: |
+        rsync --archive --del --checksum ./keras-io/tf/ "./docs/${KERAS_GUIDES_DIR}"
```

Comment:
If we are deleting `keras-io/tf` then do we need to rsync?

Reply:
Added `mkdir -p` if we delete `keras-io/tf/`.
rsync just makes it easy to delete notebooks over here if they delete notebooks over there.